R/data-storage.R

```diff
@@ -175,7 +207,7 @@ DataStorageRSQLite <- R6::R6Class( # nolint object_name_linter
       super$initialize(username, session_id)
 
       checkmate::assert_string(username)
-      logger::log_info("path to db: {db_path}")
+      logger::log_debug("path to db: {db_path}", namespace = "shiny.telemetry")
```

Comment:
Changed for consistency. No need to show this information on a production application